packages/react/src/Tabs/Tabs.tsx

```diff
@@ -25,7 +25,7 @@ const TabsRoot = forwardRef(
 
     const allTabs = useMemo(() => {
       if (!Array.isArray(children)) return []
-      return (children[0].props.children as ReactNode[]).map((child) => child)
+      return children[0].props.children
```

Comment:
Do we remember why this casting was done in the first place? (I don’t.)

Reply:
Nope, I don't know either. Removing it doesn't seem to be break anything.

Reply:
We might be bumping into React internals here. This change does indeed fix it. We discussed that we should not let `Tabs` interfere with children of `Tabs.List`; I’ll add a ticket for it.